src/cfnlint/custom_rules.py

```diff
@@ -25,16 +26,18 @@ def check(filename, template):
             if rule.valid and rule.resourceType[0] != '#':
                 try:
                     resource_properties = template.get_resource_properties([rule.resourceType])
-                    result = Operator[rule.operator](rule, resource_properties)
-                    if result != rule.operator:
-                        matches.append(cfnlint.rules.Match(line_number, '0', '0', '0', filename, CustomRule('E9999'), result, None))
+                    result = Operator[rule.operator](template, rule, resource_properties)
+                    if result.message != rule.operator:
+                        matches.append(result)
                 except KeyError:
-                    matches.append(cfnlint.rules.Match(line_number, '0', '0', '0', filename, CustomRule('E9999'),
+                    matches.append(cfnlint.rules.Match(line_number, '0', '0', '0', filename, cfnlint.customRules.Operators.CustomRule('C9999'),
                                                        str(rule.operator) + ' not in supported operators: [EQUALS] at ' + str(line), None))
```

Comment:
this error message doesn't seem to be surfacing anymore

Reply:
I'm unfamiliar which error message you are referencing here - Could I get a bit more details?

Reply:
https://github.com/aws-cloudformation/cfn-python-lint/pull/1559#discussion_r433558528

not seeing anything outputted anymore when using an invalid operator in `custom_rules.txt`:

```bash
echo 'AWS::SNS::Topic TopicName INVALID "B"' > custom_rules.txt
```

Reply:
Corrected - I had changed that to a "Cxxxx" error code during testing, which was then filtered out as that wasn't a valid error (W/E).